docs/google/devices.md

```diff
@@ -0,0 +1,194 @@
+# Google Devices
+
+This section describes the devices in Cirq for Google hardware devices and their usage.
+Since quantum hardware is an active area of research, hardware specifications and best
+practices are constantly evolving in an attempt to continuously improve performance.
+While this information should be a solid base for beginning your quantum application,
+please work with your Google sponsor to obtain the latest information on devices that
+you plan to use. 
+
+## General limitations
+
+Qubits on Google devices are laid out in a grid structure.  Connectivity is limited to
```

Comment:
nit: One space after period throughout (so says the [chicago manual of style](https://www.chicagomanualofstyle.org/qanda/data/faq/topics/OneSpaceorTwo.html), anyway :-))

Reply:
You can use two spaces in a monospaced font, as it provides visual separation between sentences. The trouble began when people stopped using typewriters and started using word processors with variable-width fonts and continued to use two spaces.

Reply:
I was always taught to use two spaces, but I changed it to one space.